### PR TITLE
[dev-v2.7] Update cis-operator imagePullPolicy

### PR DESCRIPTION
**Pull request by doflamingo721**
Updated the image pull policy to `IfNotPresent` so that if the image already exists it won't be pulled from the registry.

Depends on: https://github.com/rancher/cis-operator/pull/171
Adds:
````
cis-benchmark: 3.0.1-rc1
cis-benchmark-crd: 3.0.1-rc1
````
Linked issue: https://github.com/rancher/rancher/issues/39673
SURE: 5093

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.10
+    tag: v1.0.10-rc1
```
Comment (mitulshah-suse) [inline packages/rancher-cis-benchmark/charts/values.yaml]: The tag should be 1.0.11-rc1
1.0.10 is already released

Comment (doflamingo721) [reply packages/rancher-cis-benchmark/charts/values.yaml]: bumped the cis-operator version

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.10
+    tag: v1.0.11-rc1
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/values.yaml:8]: Why do we need to bump the cis-operator version? Is there a commit added for this change to cis-operator?

Comment (mitulshah-suse) [reply packages/rancher-cis-benchmark/charts/values.yaml:8]: Yes the pull policy for the images are updated in the cis-operator code as well.
I see you already checked and approved that.. https://github.com/rancher/cis-operator/pull/171

Comment (mitulshah-suse) [reply packages/rancher-cis-benchmark/charts/values.yaml:8]: @prachidamle Have merged the PR and created the relevant tag. Please approve.

**Review by prachidamle — DISMISSED** [review]
lgtm, needs to be rebased and resolve conflicts

**Comment by doflamingo721** [comment]
@prachidamle I have rebased and resolved the conflicts. Can you please give one final review before merging? Thanks.

**Review by prachidamle — APPROVED** [review]
LGTM

**Pull request merged**